consensus/istanbul/uptime/uptime_test.go

```diff
@@ -0,0 +1,124 @@
+package uptime
+
+import (
+	"math/big"
+	"reflect"
+	"testing"
+)
+
+func TestGetUptimeMonitoringWindow(t *testing.T) {
+	type args struct {
+		epochNumber        uint64
+		epochSize          uint64
+		lookbackWindowSize uint64
+	}
+	tests := []struct {
+		name  string
+		args  args
+		want  uint64
+		want1 uint64
+	}{
+		{"tally on first epoch", args{1, 10, 2}, 1 + 2, 9},
+		{"tally on second epoch", args{2, 10, 2}, 11 + 2, 19},
+		{"lookback window too big", args{1, 10, 10}, 11, 9},
+		// TODO: Add test cases.
```

Comment:
Is this TODO intended to be completed before finalizing the PR?  If not, I would remove it.